cli/js/doc.ts

```diff
@@ -0,0 +1,297 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

Comment:
I wish I could use this code in deno_website2 because i want to run doc generation inside the cloudflare worker... I guess it's not possible to have this code in /std ?

Reply:
Actually we could, though I would need to pass the TypeScript namespace in as a reference, so that it doesn't have presumptions about how TypeScript is exposed.  While it won't fully deno_website2, it would at least allow you to run it outside of the compiler worker.